packages/grid/src/Grid.tsx

```diff
@@ -551,6 +552,58 @@ class Grid extends PureComponent<GridProps, GridState> {
     }
   }
 
+  setFocusRow(focusedRow: number): void {
+    this.setState({ top: focusedRow - 1 }, () => this.moveFocusRow());
+    this.setState({
+      selectedRanges: [
+        new GridRange(null, focusedRow - 1, null, focusedRow - 1),
+      ],
+    });
+  }
+
+  moveFocusRow(): void {
+    if (!this.metrics || !this.prevMetrics) {
+      return;
+    }
+
+    const { top } = this.state;
+
+    const {
+      rowHeight,
+      bottom,
+      visibleRowYs,
+      visibleRowHeights,
+      columnHeaderHeight,
+      userRowHeights,
+    } = this.metrics;
+
+    const arr: [number, number][] = [];
+    userRowHeights.forEach((x, number) => arr.push([x, number]));
+    let currentTop = top;
+    const tableHeight =
+      (this.canvas?.clientHeight ?? columnHeaderHeight) - columnHeaderHeight;
```

Comment:
Just use `height` from metrics instead of `this.canvas?.clientHeight`. Also should use `gridY` instead of `columnHeaderHeight`